src/pages/en/reference/configuration-reference.md

```diff
@@ -52,7 +54,7 @@ $ astro build --project-root ./my-project-directory
 ### dist
 
 **Type:** `string`  
-**Default:** `"./dist"`
+**Default:** `"./dist"`  
```

Comment:
Nit: remove spacing changes in this file

Reply:
That's actually Markdown for `<br>`!

Reply:
But given the Markdown is generated, it might be an idea to use an explicit `<br>` rather than the extra spaces at the end of the line.

Reply:
Ah I see! Just noticed we were using this elsewhere in the docgen. Not totally sure why `\n` won't do the job though, since I'm noticing all the **Option:** `thing` get merged into the same `p` tag. Happy to keep this for consistency though

Reply:
Yeah, lines separated by `\n` get concatenated.

```md
This text
has levels.
```

Outputs:

```html
<p>This text has levels.</p>
```

But if a line ends with two or more spaces, the line break is respected as a line break.

It’s pretty sneaky behaviour and an explicit `<br>` is probably less confusing.

Reply:
Hm, that's surprising actually. Wouldn't a double newline `\n\n` resolve the issue? I know that markdown concatenates when the extra newline is missing.

Also sorry I can't test this myself. Still working on a wild `docgen` error on my machine 😆

Reply:
No worries, seems to run OK for me.

I’ve just pushed a tweak to the logic for this to use `<br>` instead. `\n\n` would break the lines up into separate paragraphs.

Reply:
Scratch that — using `<br>` or `<br />` seems to result in Markdown not outputting a `<p>` tag for that text block, so we’re back to `  ` at line endings, which produces the desired HTML output.

Reply:
Ah yep, I actually _was_ proposing that each breaks into a separate paragraph. I'll leave that to my own investigation though. Fine with this!

Reply:
After all that, I noticed there was _still_ something off in the rendering. Turns out somehow Markdown (remark? remark + Astro??) didn’t parse something as a paragraph if it contained an HTML tag, so

```md
**Type:** `string | boolean`  
**Default:** `false`
```

outputs `<p><strong>Type...`, but

```md
**Type:** `string`  
**Default:** `'localhost'`  
<Since v="0.24.0" />
```

just outputs `<strong>Type</strong>...` — no `<p>` tag. I’d guess that’s a parsing bug, but no idea where it might be, so ignoring that for now.

My solution for now is to explicitly wrap these blocks in `<p>` in the docgen script as that irons out inconsistencies. As a bonus that lets us use explicit `<br>` tags too, so… thanks weird rendering bug!